stories/erase-your-social

```diff
@@ -0,0 +1,15 @@
+To be apart of "Gen Z" comes with privilege and responsibility. With modern medicine, technology, one could argue we live
+in the best of times. The "youth", our young striving adults are often pointed at when the future is mentioned. We must
+continue the cycle to have a better tommorrow, so that the next generation has it that much easier. Or at the very least
+has the oppurtunity to do so. With it being "Gen Z's" turn how well are we stacking up to our predecessors ? I'm sure 
+we've all encountered the stubborn oldhead who swears everything was better in his/her day. We painfully listen and 
+roll our eyes but how much truth is there to be heard ? The biggest change to factor in, (my ever so humble opinion), lies within what 
```

Comment:
Whoops! It looks like you have an extra space between the word "heard" and `?`. Let's close that gap.